.test-infra/jenkins/job_LoadTests_Combine_Flink_Go.groovy

```diff
@@ -0,0 +1,136 @@
+        job_name             : "load-tests-go-flink-batch-combine-2-${now}",
+        influx_namespace     : 'flink',
+        influx_measurement   : 'go_batch_combine_2',
+        input_options        : '\'{' +
+        '"num_records": 5000000,' +
+        '"key_size": 10,' +
+        '"value_size": 90}\'',
+        fanout               : 4,
+        top_count            : 20,
+        parallelism          : 16,
+        endpoint             : 'localhost:8099',
+        environment_type     : 'DOCKER',
+        environment_config   : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+    [
+      title          : 'Group By Key Go Load test: fanout 8 times with 2GB 10-byte records total',
+      test           : 'combine',
+      runner         : CommonTestProperties.Runner.FLINK,
+      pipelineOptions: [
+        job_name             : "load-tests-go-flink-batch-combine-3-${now}",
+        influx_namespace     : 'flink',
+        influx_measurement   : 'go_batch_combine_3',
+        fanout               : 8,
+        top_count            : 20,
+        parallelism          : 16,
+        input_options        : '\'{' +
+        '"num_records": 2500000,' +
+        '"key_size": 10,' +
+        '"value_size": 90}\'',
+        endpoint             : 'localhost:8099',
+        environment_type     : 'DOCKER',
+        environment_config   : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
+      ]
+    ],
+  ].each { test -> test.pipelineOptions.putAll(additionalPipelineArgs) }
+}
+
+def loadTestJob = { scope, triggeringContext, mode ->
+  Map<Integer, List> testScenariosByParallelism = batchScenarios().groupBy { test ->
+    test.pipelineOptions.parallelism
+  }
+  Integer initialParallelism = testScenariosByParallelism.keySet().iterator().next()
+  List initialScenarios = testScenariosByParallelism.remove(initialParallelism)
+
+  def flink = new Flink(scope, "beam_LoadTests_Go_Combine_Flink_${mode.capitalize()}")
+  flink.setUp(
+      [
+        "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest"
+      ],
+      initialParallelism,
+      "${DOCKER_CONTAINER_REGISTRY}/beam_flink1.10_job_server:latest")
+
+  // Execute all scenarios connected with initial parallelism.
+  loadTestsBuilder.loadTests(scope, CommonTestProperties.SDK.GO, initialScenarios, 'combine', mode)
+
+  // Execute the rest of scenarios.
+  testScenariosByParallelism.each { parallelism, scenarios ->
+    flink.scaleCluster(parallelism)
+    loadTestsBuilder.loadTests(scope, CommonTestProperties.SDK.GO, scenarios, 'combine', mode)
+  }
+}
+
+PhraseTriggeringPostCommitBuilder.postCommitJob(
+    'beam_LoadTests_Go_Combine_Flink_Batch',
+    'Run Load Tests Go Combine Flink Batch',
+    'Load Tests Go Combine Batch suite',
+    this
+    ) {
+      additionalPipelineArgs = [:]
+      loadTestJob(delegate, CommonTestProperties.TriggeringContext.PR, 'batch')
+    }
+
+CronJobBuilder.cronJob('beam_LoadTests_Go_Combine_Flink_Batch', 'H 10 * * *', this) {
```

Comment:
We already have two tests firing at the same time (at 10 am UTC): Go ParDo (merged) and Go GBK (will be merged soon). Let's choose something else. I think `8` is a good choice.